samples/matter/common/src/events/matter_event_handler.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#pragma once
+
+#include <platform/PlatformManager.h>
+
+namespace Nordic
+{
+namespace Matter
+{
+	/**
+	 * @brief Safely register custom Matter event handler
+	 *
+	 * This function adds user provided Matter event handler in a thread safe manner.
```

Comment:
Could you copy and paste part of `DefaultEventHandler` description about what Matter event is here? I think it would be good to briefly describe what kinds of events are generated and can be handled in the given handler.

Reply:
Sure,

Reply:
I added the generic documentation bit on top of the event handler module.